src/protocols/ethereum/components/TransferReview.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <TransferReviewBase
+    :base-token-symbol="ETH_SYMBOL"
+    :transfer-data="transferData"
+    :loading="loading"
+    show-fiat
+    :protocol="PROTOCOL_ETHEREUM"
+    class="transfer-review"
+  >
+    <template #total>
+      <DetailsItem
+        :label="$t('common.total')"
+        class="details-item"
+      >
+        <template #value>
+          <TokenAmount
+            :amount="+transferData.total"
+            :symbol="ETH_SYMBOL"
+            high-precision
+            :protocol="PROTOCOL_ETHEREUM"
+            data-cy="review-total"
+          />
+        </template>
+      </DetailsItem>
+    </template>
+  </TransferReviewBase>
+</template>
+
+<script lang="ts">
+import {
+  defineComponent,
+  PropType,
+  ref,
+} from 'vue';
+import { useRouter } from 'vue-router';
+import { useI18n } from 'vue-i18n';
+import { useModals, useUi } from '@/composables';
+import type { TransferFormModel } from '@/types';
+import { PROTOCOL_ETHEREUM } from '@/constants';
+
+import TransferReviewBase from '@/popup/components/TransferSend/TransferReviewBase.vue';
+import DetailsItem from '@/popup/components/DetailsItem.vue';
+import TokenAmount from '@/popup/components/TokenAmount.vue';
+import { ETH_SYMBOL } from '@/protocols/ethereum/config';
+
+export default defineComponent({
+  name: 'EthTransferReview',
+  components: {
+    TokenAmount,
+    DetailsItem,
+    TransferReviewBase,
+  },
+  model: {
+    prop: 'transferData',
+  },
+  props: {
+    transferData: { type: Object as PropType<TransferFormModel>, required: true },
+  },
+  setup(props, { emit }) {
+    const { t } = useI18n();
+    const router = useRouter();
+    const { homeRouteName } = useUi();
+    const { openDefaultModal } = useModals();
+
+    const loading = ref<boolean>(false);
+
+    function openTransactionFailedModal(msg: string) {
+      openDefaultModal({
+        title: t('modals.transaction-failed.msg'),
+        icon: 'critical',
+        msg,
+      });
+    }
+
+    // TODO: replace that
+    // This is happening because of inner TemplateRenderer issue on parsing the blocksteam response
+    function HtmlEncode(content: string) {
+      const textAreaDiv = document.createElement('textarea');
+      textAreaDiv.textContent = content;
+      return textAreaDiv.innerHTML;
+    }
```

Comment:
The same function is used in Bitcoin TransferReview component. I suggest to move it to utilities file. Also I suggest to add a comment that describes what this function actually do.
@CedrikNikita maybe you can suggest something?

Reply:
This can be removed from the ethereum component as we won't use blockstream